membership-attribute-service/app/models/Attributes.scala

```diff
@@ -85,6 +92,11 @@ case class Attributes(
       || isGuardianPatron
   )
 
+  lazy val feastIosSubscriptionGroup =
```

Comment:
What is a subscription group?

Reply:
![image](https://github.com/guardian/members-data-api/assets/1515970/8d34d4d2-1e8e-4dae-9eb4-bb94ba4f877e)